ui/pages/swaps/index.js

```diff
@@ -372,6 +374,13 @@ export default function Swap() {
                 );
               }}
             />
+            <Route
+              path={AWAITING_SIGNATURES_ROUTE}
+              exact
+              render={() => {
+                return <AwaitingSignatures />;
```

Comment:
I will need to add some eligibility checks here in case someone tries to access this route without doing a swap with a HW wallet.